server/src/main/resources/api-docs.yaml

```diff
@@ -138,6 +152,8 @@ definitions:
       - LIVE
       - DESYNCHRONIZED
```

Comment:
I would document the relationship between this and the other values. Possibly suffix of _OTHER or _UNKNOWN

Reply:
What do you mean the other values? do you mean `REMOTE_CHANGED` and `LOCAL_CHANGED`?